sqlmesh/core/engine_adapter/redshift.py

```diff
@@ -359,7 +374,25 @@ def parse_nested() -> t.Optional[t.Dict]:
     while curr():
         nested = parse_nested()
 
-        if nested and nested.get("name") == "RESULT":
+        if nested and nested.get("name") in ("RESULT", "SEQSCAN"):
             return nested
         advance()
     return None
+
+
+# https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_type.dat
+REDSHIFT_PLAN_TYPE_MAPPINGS = {
```

Comment:
is this sufficient?

Reply:
This covers the most common Redshift types as per https://docs.aws.amazon.com/redshift/latest/dg/c_Supported_data_types.html. We'll add more later if needed